### ISSUE TITLE
[hassio-meross] Appliance.Control.Toggle

### ISSUE THREAD

**Issue by carlosatta** [carlosatta/hassio-addons#4]
Some device seams to use Toggle ability with me message "Unsupported ability Appliance.Control.Toggle for the divice xxxxxxxxxxxxxxxxxxxxxxxxxx "

**Comment by gianky69** [comment]
So seems my MSS310

**Comment by gianky69** [comment]
I’ve reinstalled your plugin 5 minute sago, but the error about MSS310 continues

**Comment by carlosatta** [comment]
@gianky69 can u contact me directly, for ex on hangout?

**Comment by gianky69** [comment]
hangout: giancarlo.massari@gmail.com

**Comment by gianky69** [comment]
I’ve tried to recall you via Hangouts, but no answer (probably busy time, but not sure about Hangouts privacy). Could you please check?

**Comment by gianky69** [comment]
Any news about MS310 v1.0 integration?

**Comment by carlosatta** [comment]
No, is very complicated for me develop the feature without the hardware.

If you ship the socket to me, I will implement the feature and give it back to you in a couple of week.
